runtime/src/types.rs

```diff
@@ -271,3 +268,69 @@ mod h256_le {
         }
     }
 }
+
+mod dispatch_error {
```

Comment:
Uf this is quite ugly. Did you open an issue on subxt to see if it's possible to allow a more elegant solution? Also, how robust is this module? Will we get a compilation error if the dispatcherror type ever changes?

Reply:
https://github.com/paritytech/subxt/issues/426#issuecomment-1028917908